.github/workflows/stale_marker.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Close stale issues and PR'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 7 days.'
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 7 days with no activity.'
+          days-before-issue-stale: 45
+          days-before-pr-stale: 60
+          days-before-issue-close: 7
+          days-before-pr-close: -1
```

Comment:
what does it mean?

Reply:
Stale PRs would not be closed

Reply:
what is it for then?

Reply:
Mark it as stale. By default, all stale issues and PRs are closed.

Reply:
yours statements contradict each other.

Reply:
I mean, if not specify explicitly `days-before-pr-close: -1`, PR will be closed after default timeout period.